src/Microsoft.VisualStudio.Composition/Configuration/AttributedPartDiscovery.cs

```diff
@@ -333,7 +333,7 @@ private bool TryCreateImportDefinition(Type importingType, ICustomAttributeProvi
             }
             else if (importManyAttribute != null)
             {
-                this.ThrowOnInvalidImportingMemberOrParameter(member);
+                this.ThrowOnInvalidImportingMemberOrParameter(member, isImportMany: true);
```

Comment:
I think it would make more sense just to omit the call in this case since setting `isImportMany` to true makes the function do nothing.

Reply:
The fact that it does nothing is an implementation detail of the method. In the future we may add more validations to importing members that may apply to ImportMany.